app/realtime.php

```diff
@@ -519,6 +519,12 @@ function getCache(): Cache
             ]
         ];
 
+        // Handle SQL error code is 'HY000'
+        $code = $th->getCode();
+        if (!is_int($code)) {
+            $code = 500;
+        }
+
         $server->send([$connection], json_encode($response));
         $server->close($connection, $th->getCode());
```

Comment:
```suggestion
        $server->close($connection, $code);
```

Reply:
Oops! That was my intention but I forgot to replace this 🙈